ext/templates-modals.html

```diff
@@ -298,6 +298,10 @@ <h5>or click here to upload</h5>
             </div>
         </div>
         <div class="modal-body">
+            <p>
```

Comment:
What would a <small> tag look like?

Reply:
> What would a tag look like?

Exmaple of small tag
![image](https://github.com/user-attachments/assets/3af8de12-6f27-4c3f-b3e3-9a85711675ae)

Reply:
Looks the same haha